src/cpp_common/undirectedHasCostBG.cpp

```
@@ -26,13 +26,13 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
  ********************************************************************PGR-GNU*/
 
 #include "cpp_common/undirectedHasCostBG.hpp"
-
+#include <stdexcept>
 #include <utility>
 #include <vector>
 #include <deque>
 #include <string>
+#include <cassert>
```

Cppcheck (2.20.0)

[information] Include file

(missingIncludeSystem)